SceneComponent.js

```diff
@@ -0,0 +1,19 @@
+const React = require('react');
+const ReactNative = require('react-native');
+const {Component, } = React;
+const {View, StyleSheet, } = ReactNative;
+
+const StaticContainer = require('react-static-container');
+
+const SceneComponent = (Props) => {
+  const {selected, ...props, } = Props;
+  return(
```

Comment:
https://github.com/Khan/style-guides/blob/master/style/react.md#open-elements-on-the-same-line